content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -120,6 +134,26 @@ The default configuration can be enabled by setting `--enable-leader-migration`
 
 For `kube-controller-manager` and `cloud-controller-manager`, if there are no flags that enable any in-tree cloud provider or change ownership of controllers, the default configuration can be used to avoid manual creation of the configuration file.
 
+### Special Case: Migrating Node IPAM Controller
```

Comment:
```suggestion
### Special case: migrating the Node IPAM controller {#node-ipam-controller-migration}
```

Reply:
That's a much better anchor name. Thank you! Applied.